client/components/header-cake/back.jsx

```diff
@@ -39,7 +41,7 @@ export default React.createClass( {
 
 	render() {
 		const { text = this.translate( 'Back' ), href, onClick, spacer } = this.props;
-		const windowWidth = window.innerWidth;
+		const windowWidth = global.window.innerWidth || 1024;
```

Comment:
It's a magic number, I'm glad you picked up on this :) 

Given that we can't detect window size on the server AFAICS, we should try to set something sane here, so the back text doesn't erroneously appear in the initial render.

I was thinking that if we could semi-reliably check whether a device is mobile, or a desktop, based on the UA — maybe with [`express-device`](https://github.com/rguerreiro/express-device) — we could specify some sane defaults, e.g. something over 768 for desktop, and something under that for mobile. This could be factored into `lib/viewport`.